codecov.yaml

```diff
@@ -1,53 +1,74 @@
 codecov:
+  # The default branch is used to identify:
+  #  Which branch to cache the repository YAML for UI changes.
+  #  Which branch is the first branch on the repository dashboard in Codecov.
   branch: main
+  # Show the Codecov status without waiting for other status checks to complete or pass.
   require_ci_to_pass: no
 
-comment:
-  layout: "reach,diff,flags,tree"
-  behavior: default
-  require_changes: no
-  require_base: no
-  require_head: no
-  after_n_builds: 1
-  show_carryforward_flags: true
+comment: false
+#  # Customize comment layout by choosing the order and types of components to include.
```

Comment:
are we keeping this whole comment block on purpose?

Reply:
It seems that the test coverage data in the comments are not accurate and cause misunderstanding often. Thus we had better block the codecov comment and refer to the status check results.

Reply:
I was asking about the codecov.yaml file itself, and whether you wanted to keep lines 10 through 35, which are commented out

Reply:
Oh, thanks for the reminder. Keeping this whole comment block in case we need to restore this modification or refer to the comment config. Seems it’s not a typical practice in other projects? I will update this.